packages/react-kit/src/components/test.page.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { PureComponent } from 'react';
+import { withRX } from '../utils/with-rx';
+import { interval } from 'rxjs';
+import { storiesOf } from '@storybook/react';
+
+type FooProps = {
+	foo: string;
+	bar: number;
+};
+class Foo extends PureComponent<FooProps> {
+	render() {
+		const { foo, bar } = this.props;
+		return (
+			<div>
+				{foo}, {bar}
+			</div>
+		);
+	}
+}
+
+const FooContainer = withRX(
+	Foo,
+	(props$, select) =>
```

Comment:
I think we should have more examples, for now new signature confusing me a bit:
- how/when I should use props$ argument
- how I can use effects$ stream